src/oatpp/core/utils/ConversionUtils.cpp

```diff
@@ -189,11 +190,30 @@ namespace oatpp { namespace utils { namespace conversion {
   }
 
   v_buff_size float32ToCharSequence(v_float32 value, p_char8 data, v_buff_size n, const char* format) {
-    return snprintf((char*)data, n, format, value);
+    return float64ToCharSequence(value, data, n, format);
   }
 
-v_buff_size float64ToCharSequence(v_float64 value, p_char8 data, v_buff_size n, const char* format) {
-    return snprintf((char*)data, n, format, value);
+  v_buff_size float64ToCharSequence(v_float64 value, p_char8 data, v_buff_size n, const char* format) {
+    const v_buff_size ret = snprintf((char*)data, n, format, value);
+
+    const char locale_decimal_point = localeconv()->decimal_point[0];
+    static constexpr char json_decimal_point = '.';
+    if (locale_decimal_point != json_decimal_point) {
+      convertSingleDecimalPointInCharSequence(data, locale_decimal_point, json_decimal_point);
+    }
+
+    return ret;
+  }
+
+  p_char8 convertSingleDecimalPointInCharSequence(p_char8 data, char old_char, char new_char) {
```

Comment:
Please move this method to `json::Utils` - 
https://github.com/oatpp/oatpp/blob/master/src/oatpp/parser/json/Utils.hpp